library/Messages/tr_TR.php

```diff
@@ -0,0 +1,954 @@
+<?php
+namespace Respect\Validation;
+
+use Respect\Validation\Exceptions\ValidationException;
+use Respect\Validation\Exceptions\AbstractGroupedException;
+use Respect\Validation\Exceptions\AlphaException;
+use Respect\Validation\Exceptions\AlwaysInvalidException;
+use Respect\Validation\Exceptions\AttributeException;
+use Respect\Validation\Exceptions\BetweenException;
+use Respect\Validation\Exceptions\DateException;
+use Respect\Validation\Exceptions\EqualsException;
+use Respect\Validation\Exceptions\IpException;
+use Respect\Validation\Exceptions\LengthException;
+use Respect\Validation\Exceptions\MaxException;
+use Respect\Validation\Exceptions\MinException;
+use Respect\Validation\Exceptions\NotEmptyException;
+
+$messages = array(
```

Comment:
> You could just return the array instead of creating a variable it would be much more clean.

I would argue that the best approach would be keeping the messages inside the rule and translation them while creating the exceptions.